### PR TITLE
Fix mdspell assertion failure in CHANGELOG.md

### PR DESCRIPTION
**Pull request by rjmholt**
## PR Summary

`mdspell` doesn't like the unordered list indentation in the changelog. This makes CHANGELOG.md comply with `mdspell`'s rules. 

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 - Disallow Basic Auth over HTTP in PowerShell Remoting on Unix (#6787)
 - Change packaging to differentiate only between major versions and previews (#6968)
 - Enhance and refactor `Add-Type` cmdlet (#6141) (Thanks @iSazonov!)
-  - A few error strings were removed and thus the corresponding fully-qualified error ids are not in use anymore.
+    + A few error strings were removed and thus the corresponding fully-qualified error ids are not in use anymore.
```
Comment (daxian-dbw) [inline CHANGELOG.md]: Please use `    -` (4 spaces plus minus) to fix it. This is to keep consistent with the existing instances in this file.

Comment (rjmholt) [reply CHANGELOG.md]: Ah I read the wrong rule in https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md.

Thought `mdspell` was complaining about [MD004](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md004---unordered-list-style).

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
> it dumped out a number of errors

@rjmholt are you running `mdspell` locally? I don't see any errors when running `mdspell "**/*.md" --ignore-numbers --ignore-acronyms` (see below).

Yeah, the locale, we should use `en-us` explicitly.

```
bash:228> mdspell "**/*.md" --ignore-numbers --ignore-acronyms
Spelling - assets/README.md
Spelling - CHANGELOG.md
Spelling - CODE_OF_CONDUCT.md
Spelling - demos/Apache/readme.md
Spelling - demos/Azure/README.md
Spelling - demos/crontab/README.md
Spelling - demos/DSC/readme.md
Spelling - demos/powershellget/README.md
Spelling - demos/python/README.md
Spelling - demos/README.md
Spelling - demos/rest/README.md
Spelling - demos/SystemD/readme.md
Spelling - demos/WindowsPowerShellModules/README.md
Spelling - docker/README.md
Spelling - docker/tests/README.md
Spelling - docs/building/internals.md
Spelling - docs/building/linux.md
Spelling - docs/building/macos.md
Spelling - docs/building/windows-core.md
Spelling - docs/cmdlet-example/command-line-simple-example.md
Spelling - docs/cmdlet-example/visual-studio-simple-example.md
Spelling - docs/community/governance.md
Spelling - docs/debugging/README.md
Spelling - docs/dev-process/breaking-change-contract.md
Spelling - docs/dev-process/coding-guidelines.md
Spelling - docs/dev-process/resx-files.md
Spelling - docs/DOCSMIGRATION.md
Spelling - docs/FAQ.md
Spelling - docs/git/basics.md
Spelling - docs/git/README.md
Spelling - docs/git/submodules.md
Spelling - docs/host-powershell/README.md
Spelling - docs/learning-powershell/create-powershell-scripts.md
Spelling - docs/learning-powershell/debugging-from-commandline.md
Spelling - docs/learning-powershell/powershell-beginners-guide.md
Spelling - docs/learning-powershell/README.md
Spelling - docs/learning-powershell/using-vscode.md
Spelling - docs/learning-powershell/working-with-powershell-objects.md
Spelling - docs/maintainers/issue-management.md
Spelling - docs/maintainers/README.md
Spelling - docs/maintainers/releasing.md
Spelling - docs/testing-guidelines/CodeCoverageAnalysis.md
Spelling - docs/testing-guidelines/getting-code-coverage.md
Spelling - docs/testing-guidelines/PowerShellCoreTestStatus.md
Spelling - docs/testing-guidelines/testing-guidelines.md
Spelling - docs/testing-guidelines/TestRoadmap.md
Spelling - docs/testing-guidelines/WritingPesterTests.md
Spelling - README.md
Spelling - src/libpsl-native/README.md
Spelling - src/Microsoft.PowerShell.SDK/README.md
Spelling - src/Modules/README.md
Spelling - src/powershell/README.md
Spelling - src/TypeCatalogGen/README.md
Spelling - test/csharp/README.md
Spelling - test/powershell/README.md
Spelling - test/README.md
Spelling - test/tools/CodeCoverageAutomation/README.md
Spelling - test/tools/Modules/WebListener/README.md
Spelling - test/tools/WebListener/README.md
Spelling - tools/install-powershell-readme.md
Spelling - tools/releaseBuild/README.md
>> 61 files are free from spelling errors
```

**Comment by rjmholt** [comment]
I was running this:
https://github.com/PowerShell/PowerShell/blob/6ea1389571d227a148e0dcc57d7843f7179bd20a/.travis.yml#L49

And got this output:
```
> mdspell '**/*.md' '!**/Pester/**/*.md' --ignore-numbers --ignore-acronyms --report
    docs/git/basics.md
       37 | references to our favorites.
       49 | #### Katacoda
       52 | [Git lessons on katacoda](https://www.katacoda.com/cou
       54 | #### Githug
       56 | [Githug](https://github.com/Gazler/gi

    docs/git/README.md
       30 | * Use lowercase-with-dashes for naming.

    docs/learning-powershell/powershell-beginners-guide.md
       44 | interested in the instance of firefox process that are running on y
      134 | ld!" as a file content to the test.txt.
      135 | Because the test.txt file exists already, we use *
      173 | **8. del - Remove-Item**: Deletes the
      175 | ill delete the file /home/jen/test.txt:
      180 | **9. $PSVersionTable**: Displays the version of Po
      182 | Type **$PSVersionTable** in your PowerShell session,
      271 | th PowerShell][remoting] from Channel9
      272 | - [eBooks from PowerShell.org](https://
      272 | - [eBooks from PowerShell.org](https://powershell.org/ebook
      273 | - [eBooks from PowerShell.com][ebooks-p
      273 | - [eBooks from PowerShell.com][ebooks-powershell.com]
      274 | - [eBooks List][ebook-list] by Martin S
      274 | s List][ebook-list] by Martin Schvartzman
      279 | ting in Depth][in-depth] from Channel9
      280 | Management][remote-mgmt] from ITPro

    docs/learning-powershell/README.md
       91 | | pwd                 |pwd, $pwd, G
       91 | | pwd                 |pwd, $pwd, Get-Location      |Sho
       91 | | pwd                 |pwd, $pwd, Get-Location      |Show work
       92 | | clear, Ctrl+L, reset| cls, clear                  |Clea
       93 | | mkdir               |New-Item -Item
       93 | kdir               |New-Item -ItemType Directory |Create a new folde
       94 | | touch test.txt      |New-Item -Path test.txt
       94 | test.txt      |New-Item -Path test.txt      |Create a new empty file
       95 | | cat test1.txt test2.txt         |Get-Conten
       95 | | cat test1.txt test2.txt         |Get-Content test1.tx
       95 | est2.txt         |Get-Content test1.txt, test2.txt       |Display fil
       95 |       |Get-Content test1.txt, test2.txt       |Display files contents
       96 | | cp ./source.txt ./dest/dest.txt
       96 | | cp ./source.txt ./dest/dest.txt |Copy-Item so
       96 | | cp ./source.txt ./dest/dest.txt |Copy-Item source.tx
       96 | | cp ./source.txt ./dest/dest.txt |Copy-Item source.txt dest/de
       96 | xt ./dest/dest.txt |Copy-Item source.txt dest/dest.txt     |Copy a fil
       96 | est.txt |Copy-Item source.txt dest/dest.txt     |Copy a file
       96 | xt |Copy-Item source.txt dest/dest.txt     |Copy a file
       97 | | cp -r ./source ./dest
       97 | | cp -r ./source ./dest           |Copy-Item ./source
       97 |         |Copy-Item ./source ./dest -Recurse     |Recursively cop
       98 | | mv ./source.txt ./dest/dest.txt
       98 | | mv ./source.txt ./dest/dest.txt |Move-Item ./
       98 | | mv ./source.txt ./dest/dest.txt |Move-Item ./source.
       98 | | mv ./source.txt ./dest/dest.txt |Move-Item ./source.txt ./des
       98 |  ./dest/dest.txt |Move-Item ./source.txt ./dest/dest.txt |Move a file
       98 | txt |Move-Item ./source.txt ./dest/dest.txt |Move a file to othe
       98 | Move-Item ./source.txt ./dest/dest.txt |Move a file to other folder
       99 | | rm test.txt
       99 | | rm test.txt                     |Remove-I
       99 |                  |Remove-Item test.txt                   |Delete a f
      100 | | rm -r &lt;folderName>
      100 | | rm -r &lt;folderName>           |Remove-Item &lt;f
      100 | e>           |Remove-Item &lt;folderName> -Recurse   |Delete a folder
      102 | | grep -Rin "sometext" --include="*.
      102 | | grep -Rin "sometext" --include="*.cs" |
      102 | | grep -Rin "sometext" --include="*.cs" |Get-ChildI
      102 | r> \| Select-String -Pattern "sometext" | Recursively case-insensiti

    docs/learning-powershell/working-with-powershell-objects.md
       64 | To confirm, we can call the GetType() method interactively from t

>> 62 spelling errors found in 61 files
```

**Pull request merged**